core/camel-xml-io/pom.xml

```diff
@@ -44,6 +44,11 @@
             <groupId>org.apache.camel</groupId>
             <artifactId>camel-xml-io-util</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Do you know why camel-xml-io needs those JAXB annotations?

Reply:
It complains about this:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project camel-xml-io: Compilation failure: Compilation failure: 
[ERROR] /home/opiske/code/java/camel/core/camel-xml-io/src/generated/java/org/apache/camel/xml/in/ModelParser.java:[28,23] error: package javax.annotation does not exist
[ERROR] /home/opiske/code/java/camel/core/camel-xml-io/src/generated/java/org/apache/camel/xml/in/ModelParser.java:[43,1] error: cannot find symbol
[ERROR]   symbol: class Generated
```

Reply:
Argh its maybe overkill to have this JAR because of a generated annotation that bring not much value

Reply:
Yeah, it seems overkill. Let me take a look and see if I can modify the package plugin to avoid using it. I will send it as a separate PR, so we don't mix things.

Reply:
FYI: https://github.com/apache/camel/pull/6677